Updated the response for api endpoints & added additional info (#8508)
magento/devdocs: src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md

The sample data provides multiple attribute sets, including Default, Top, and Bottom. To assign the Top attribute set to the product, we need to know the corresponding `attribute_set_id`.

> Use the following call to search for the attribute set named `Top` and entity_type_id equals to 4.

Reviewer: ```suggestion
Use the following call to search for the attribute set named `Top` and `entity_type_id` equals to `4`.
```

> By default, the `catalog_product` entity has the entity_type_id value as 4.

Reviewer: ```suggestion
By default, the `catalog_product` entity has the `entity_type_id` value as `4`.
```

Author: @BarnyShergold Thanks for the review. I have committed the changes.

**Endpoint:**

```html
GET <host>/rest/default/V1/products/attribute-sets/9/attributes
```

**Response:**

```json
[
    {
        "attribute_id": 141,
        "attribute_code": "size",
        "frontend_input": "select",
        "entity_type_id": "4",
        "is_required": false,
        "options": [
            {
                "label": " ",
                "value": ""
            },
            {
                "label": "Small",
                "value": "168"
            },
            {
                "label": "Medium",
                "value": "169"
            },
            {
                "label": "Large",
                "value": "170"
            }
        ],
        "is_user_defined": true,
        "default_frontend_label": "Size",
        "frontend_labels": [],
        "backend_type": "int",
        "source_model": "Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Table",
        "default_value": "",
        "is_unique": "0",
        "validation_rules": []
    }
]
```

> The response contains almost 3,000 lines and above is the little bit of response from the endpoint. The following table provides a summary of the attributes that are relevant in this tutorial.

Reviewer: ```suggestion
The response contains almost 3,000 lines and the above is a part of the response from the endpoint. The following table provides a summary of the attributes that are relevant in this tutorial.
```
